infrastructure/src/main/java/org/corfudb/infrastructure/FailureHandlerDispatcher.java

```diff
@@ -205,7 +238,10 @@ private void reconfigureSequencerServers(CorfuRuntime runtime, Layout originalLa
                 verifyStreamTailsMap(streamTails);
 
                 // Configuring the new sequencer.
-                newLayout.getSequencer(0).reset(maxTokenRequested + 1, streamTails).get();
```

Comment:
You are right. But since there wont be any new writes after paxos until we mark the new sequencer as ready as there is a difference in epoch, don't you think we can fetch tails without the quorum predicate counting?